src/blog/2022-12-24-rum-archive-benfords-law.md

```diff
@@ -0,0 +1,131 @@
+The Akamai mPulse team studies Real User Monitoring (RUM) data to help our customers understand how to optimize website performance and how user behavior may be altered by changes to configurations or resource requests on their site.  As a data scientist on the mPulse team, I search for patterns in our data and identify problem areas on customer websites in order to recommend actional improvements toward the end user experience.
```

Comment:
I'm thinking we should have a sub-folder for each post of images.  So instead of `img/*.png` maybe `[2023-01-24-rum-archive-benfords-law/*.png`?